services/ui-src/src/utils/reports/entities.ts

```diff
@@ -1,9 +1,27 @@
 import { EntityShape, OverlayModalEntityTypes, AnyObject } from "types";
 
 const getRadioValue = (entity: EntityShape | undefined, label: string) => {
-  return entity?.[label].value;
+  return entity?.[label]?.[0].value !== "Other, specify"
```

Comment:
[nit, non-blocking]: would it be worthwhile to define a variable for `entity?.[label]?.[0].value` such as 
`const radioValue = entity?.[label]?.[0].value`? It might make it easier to read this code? 🤔

Reply:
yeah, I could see `radioLabelValue` being a good name to use. This function is largely lifted from MCR so we didn't do much to change it.